src/components/List/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Card from '../Card/index.js';
```

Comment:
it's more conventional to do `import Card from '../Card'`. the `index.js` part is automatically looked for